src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Fixtures/php/mailer_with_dsn.php

```diff
@@ -13,6 +13,7 @@
             'envelope' => [
                 'sender' => 'sender@example.org',
                 'recipients' => ['redirected@example.org'],
+                'allowed_recipients' => ['/^foobar@example.org$/'],
```

Comment:
same in all examples:
```suggestion
                'allowed_recipients' => ['foobar@example\.org'],
```